src/odemis/gui/conf/test/conf_test.py

```diff
@@ -138,6 +150,14 @@ def test_save(self):
         conf.last_path = u"/home/booo/"
         conf.last_format = "HDF5"
         conf.last_extension = ".h5"
+        conf.fn_ptn = u"{timelng}-test {cnt}"
+
+    def test_save_unicode(self):
+        conf = gui.conf.get_acqui_conf()
+        conf.last_path = u"/home/boooµ/"
+        conf.last_format = "HDF5"
+        conf.last_extension = ".h5"
+        conf.fn_ptn = u"{timelng}-test {cnt} µm value"
 
```

Comment:
what is here the feedback if the test was successful?

Reply:
Actually, the attributes are properties, so setting them will call a function, which will fail if something goes wrong, and eventually the test case will fail.
I've added a comment explaining this.

Reply:
ah ok. Does it make sense to add this `# Will fail if setting the properties goes wrong` to both testcases?